code/modules/clothing/head/garlands.dm

```diff
@@ -0,0 +1,43 @@
+/obj/item/clothing/head/garland
+	name = "Floral Garland"
+	desc = "Someone, somewhere, is starving and wearing this. And its defiantly not you."
+	icon_state = "garland"
+	worn_icon_state = "garland"
+
+/obj/item/clothing/head/garland/equipped(mob/user, slot)
+	. = ..()
+	if(slot_flags & slot)
+		var/datum/component/mood/our_mood = user.GetComponent(/datum/component/mood)
+		if(our_mood?.sanity <= SANITY_NEUTRAL)
+			START_PROCESSING(SSobj, src)
+			RegisterSignal(src, COMSIG_ATOM_FIRE_ACT, .proc/stop_sanity_boost)
+
+/obj/item/clothing/head/garland/dropped(mob/user)
+	. = ..()
+	stop_sanity_boost()
+
+/obj/item/clothing/head/garland/Destroy()
+	stop_sanity_boost()
+	return ..()
+
+/// When equipped, the garland will slowly restore sanity up to neutral of non-unstable characters, as it does in Don't Starve.
+/obj/item/clothing/head/garland/process(delta_time)
+	var/mob/living/carbon/carbon_loc = loc
+	if(!istype(carbon_loc))
+		stop_sanity_boost()
+		return
+
+	// Under no circumstances do I recommend the use of GetComponent in this way, for anyone thinking about copy+pasting.
+	var/datum/component/mood/our_mood = carbon_loc.GetComponent(/datum/component/mood)
+	if(our_mood?.sanity <= SANITY_NEUTRAL + 2)
+		our_mood.setSanity(our_mood.sanity + 0.05 * delta_time, SANITY_UNSTABLE, SANITY_NEUTRAL)
```

Comment:
how does this interact with the unstable quirk?

Reply:
An earlier comment denotes that it shouldn't affect those with unstable

Reply:
Tested it.

This does not work with people who are unstable, as intended.